wiki/People/Tournament_Committee/en.md

```diff
@@ -0,0 +1,46 @@
+---
+stub: true
+---
+
+# Tournament Committee
+
+The **Tournament Committee** is a panel of skilled and established organizers or helpers in the tournament scene. 
+
+Their responsibilities include:
+
+- Managing reports from the [tournament reports form](https://pif.ephemeral.ink/tournament-reports)
+- Observing the general tournament scene and advising the account support team on a case-by-case basis.
+- Regularly reviewing the [Official tournament support](/wiki/Tournaments/Official_support) page for revisions where necessary. 
+
+The panel was first announced to the public on July 22, 2021 through the `#tournaments` channel in the [osu!dev Discord server](https://discord.com/invite/ppy) in response to the rapidly growing community tournament scene in osu!.
+
+### How can I help contribute?
+
+You can help the tournament committee by continuing to run excellent tournaments! If you have any issues or concerns regarding any aspect of a community tournament, we encourage you to bring these to our attention via the tournament reports form.
+
+Those who are interested in joining the tournament committee may express their interest to us by sending an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh). Successful candidates are expected to have a significant breadth of experience in the tournament scene, either as a player or staff. 
+
+## Members
+
+- ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578)
+- ![][flag_US] [ChillierPear](https://osu.ppy.sh/users/9501251)
+- ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779)
+- ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656)
+- ![][flag_GB] [mangomizer](https://osu.ppy.sh/users/1893718)
+- ![][flag_FR] [ThePooN](https://osu.ppy.sh/users/718454)
+- ![][flag_US] [this1neguy](https://osu.ppy.sh/users/1797189)
+- ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416)
```

Comment:
is Walter still active?

Reply:
He still weighs in on a few discussion points, yes